borrowings/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+from books.models import Book
+from .models import Borrowing
+
+
+class BookSerializer(serializers.ModelSerializer):
```

Comment:
you should follow DRY and don't repeat already created code

you can use:

from books.serializers import BookSerializer